Sources/KSCrashRecording/Monitors/KSCrashMonitor_Signal.c

```diff
@@ -96,7 +101,8 @@ static void handleSignal(int sigNum, siginfo_t* signalInfo, void* userContext)
 
         KSCrash_MonitorContext* crashContext = &g_monitorContext;
         memset(crashContext, 0, sizeof(*crashContext));
-        crashContext->crashType = KSCrashMonitorTypeSignal;
+        crashContext->monitorName = name();
+        crashContext->monitorProperties = properties();
```

Comment:
It's a bit annoying that you have to fill these two manually. 
What if you add a helper method that fills these taking the context and monitorAPI?